circle.yml

```diff
@@ -4,6 +4,7 @@ machine:
     version: oraclejdk8
   environment:
     TERM: dumb  # don't let gradle use fancy ansi seq code
+    BUILD_IMAGE: danieln/digdag-build:20160823085939Z-4c3c94d6925e4e476fca04260ec6be6352351de2
```

Comment:
Yeah, and it should be `digdag/build` instead of `danieln/digdag-build`. Created a digdag docker hub account we can use for that.